web/packages/teleport/src/Player/Player.tsx

```diff
@@ -42,16 +42,40 @@ export function Player() {
   const { sid, clusterId } = useParams<UrlPlayerParams>();
   const { search } = useLocation();
 
+  useEffect(() => {
```

Comment:
Somewhat unrelated, but this probably should have been a `useEffect` all along rather than updating the document title on every render.